src/config.ts

```diff
@@ -209,4 +209,6 @@ export default {
   VORTEX_API_BASE,
   VORTEX_APP_ID,
   VORTEX_TOKEN,
+  GRAPHQL_UPLOAD_MAX_FILE_SIZE_IN_BYTES: 5 * 1000 * 1000, // 5 MB
+  GRAPHQL_UPLOAD_MAX_FILES: 1,
```

Comment:
**suggestion**: Looks like we're not always consistent with alphabetizing these lines but we could move these two lines right before `GRAVITY_API_BASE`.